[WIP] Terraform Docs for TF policy (#2690)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/workspaces/policy-enforcement/test/index.mdx

--- 

> Page_title: Test policies

Reviewer: ```suggestion
---
page_title: Test policies
```
Something weird happening here with the page meta. It might be at least part of the reason the build isn't working?

> Description: Policy tests are a feature of Sentinel and Terraform policy that that you can use to verify that your policies work as expected before you enforce them with HCP Terraform. Learn how policy tests can help you deploy and update policies across your organization.

Reviewer: ```suggestion
description: Learn how to test policies written for Sentinel and the Terraform policy framework so that you can verify that your policies work as expected before you enforce them with HCP Terraform.
```

---

# Test policies

> Policy tests are a feature of Sentinel and Terraform policy that that you can use to verify that your policies work as expected before you enforce them with HCP Terraform. Learn how policy tests can help you deploy and update policies across your organization.

Reviewer: ```suggestion
You can write tests for policies written for the Sentinel and Terraform policy framework. Policy tests help you verify that your policies work as expected before you enforce them with HCP Terraform. Learn how policy tests can help you deploy and update policies across your organization.
```
